src/main/java/org/peergos/blockstore/FilteredBlockstore.java

```diff
@@ -0,0 +1,9 @@
+package org.peergos.blockstore;
+
+import io.ipfs.cid.Cid;
+import java.util.concurrent.CompletableFuture;
+
+public interface FilteredBlockstore extends Blockstore {
```

Comment:
FilteredBlockstore would just be another type of blockstore not an interface

Reply:
done